image-processing/linefinder.py

```diff
@@ -354,18 +219,19 @@ def plot_nice(self, name):
 
             OUTPUTS: a cleaner looking plot than that given by the other functions
         '''
-        
-        blurred = linefinder.blur_sample_gauss(self, False)
-        x = linefinder.find_lines_with_exclusions(self, view_plot=False, distance=10, min_prominences=4)
+        x = self.find_lines_with_exclusions(view_plot=False, distance=10, min_prominences=4)
 
         fig, ax = plt.subplots(ncols = 2, nrows = 1)
-        fig.suptitle('Results for {}'.format(name))
+        fig.suptitle('Results for {}:'.format(name))
+
+        txt = f"Score is 'some score' out of 10"   #need to replace this with inv_out_of_10
```

Comment:
i think i've got a plan for adding this in!

Reply:
done! (TF)